app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -90,9 +91,20 @@ def refresh_targets_for_ems(ems, targets)
 
           Benchmark.realtime_block(:save_inventory) { save_inventory(ems, target, hashes) }
           _log.info "#{log_header} Refreshing target #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+
+          if hashes.kind_of?(Array)
```

Comment:
I really dislike checking if hashes is an array to imply manager_refresh, we already do that here [save_ems_inventory](https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory.rb#L3) and I don't want to add another.